saleor/settings.py

```diff
@@ -557,6 +557,9 @@ def get_host():
         "if ENABLE_ACCOUNT_CONFIRMATION_BY_EMAIL is enabled."
     )
 
+WEBHOOK_TIMEOUT = 10
+WEBHOOK_SYNC_TIMEOUT = 20
```

Comment:
short comment might come in handy, it might be unclear for users why there are 2 settings and how they differ.